builtin/game/chatcommands.lua

```diff
@@ -677,6 +677,38 @@ core.register_chatcommand("spawnentity", {
 	end,
 })
 
+core.register_chatcommand("setnode", {
+	params = "<NodeName> [<X>,<Y>,<Z>]",
+	description = "Places node at given (or your) position",
+	privs = {interact = true, server = true},
+	func = function(name, param)
+		local nodename, pos = string.match(param, "^([^ ]+) *(.*)$")
+		if not nodename then
+			return false, "Node name required"
+		end
+		core.log("action", ("%s invokes /placenode, nodename=%q")
+				:format(name, nodename))
+		local player = core.get_player_by_name(name)
+		if not player then
+			core.log("error", "Unable to spawn node, player is nil")
+			return false, "Unable to spawn node, no player"
```

Comment:
'spawn' is the wrong word, 'spawn' implies an entity.
Should be 'place' to match the rest of the code.